runtime_util/runtime_util.h

```diff
@@ -74,6 +75,16 @@ typedef enum gp_joysticks {
     JOYSTICK_RIGHT_Y
 } gp_joystick_t;
```

Comment:
Yes this is really annoying, but could we list out enums for all 47 buttons in the same order they appear in that key list? That way we can refer to the buttons with their names as enums in tests and CLI's, instead of doing things like `(1 << 2);  <-- 'z ` in our code?

Reply:
hmmmm sounds really bad. It's really ugly to use an enum for that. I'd rather use a function (same for gamepad) that uses our predefined arrays.

Reply:
ok i added some functions that are much nicer